explorer/src/pages/Module/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Module as ModuleProps } from "@verax-attestation-registry/verax-sdk/lib/types/.graphclient";
+import { useParams } from "react-router-dom";
+import useSWR from "swr";
+
+import { Back } from "@/components/Back";
+import { NotFoundPage } from "@/components/NotFoundPage";
+import { EMPTY_STRING, links } from "@/constants";
+import { SWRKeys } from "@/interfaces/swr/enum";
+import { useNetworkContext } from "@/providers/network-provider/context";
+
+export const Module = () => {
+  const { id } = useParams();
+  const { sdk } = useNetworkContext();
+
+  const {
+    data: module,
+    isLoading,
+    isValidating,
+  } = useSWR(SWRKeys.GET_SCHEMA_BY_ID, () => sdk.module.findOneById(id || EMPTY_STRING) as Promise<ModuleProps>, {
+    shouldRetryOnError: false,
+    revalidateOnFocus: false,
+  });
+
+  //todo add loading
```

Comment:
Let's add a placeholder Loading component to get rid of all the todos related to it across all files.